### ISSUE TITLE
Escaping of attributes

### ISSUE THREAD

**Issue by groe** [ohler55/ox#149]
I have noticed that `Ox::Builder#element(name, attributes)` does not escape attribute values.

```ruby
builder = Ox::Builder.new
builder.element(:div, title: '"><script>alert("hacked");</script>')
builder.pop
builder.to_s
# <div title=""><script>alert("hacked");</script>"/>
```

Is this a bug or a deliberate design decision (meaning users of ox should take care of escaping everything before passing it to the builder)?

The `Ox::Builder#text(str)` method seems to escape the `str` passed to it.

**Comment by ohler55** [comment]
The attribute should be escaped. It is a bug. I'll get it fixed.

**Comment by ohler55** [comment]
I pushed a fix onto github. Let me know if it works for you please.

**Comment by groe** [comment]
Thanks @ohler55! I will try and let you know.

**Comment by groe** [comment]
Works for me!

**Comment by ohler55** [comment]
Great, just released 2.4.4.
